src/main/java/runge/weather/CurrentWeatherFrame.java

```diff
@@ -0,0 +1,75 @@
+package runge.weather;
+
+import io.reactivex.rxjava3.disposables.Disposable;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava3.RxJava3CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class CurrentWeatherFrame extends JFrame
+{
+    CurrentWeatherView currentWeatherView;
+    WeatherService weatherService;
+
+    public CurrentWeatherFrame() {
+        setSize(780, 500);
+        setTitle("Current Weather");
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        JPanel mainPanel = new JPanel();
+        mainPanel.setLayout(new BorderLayout());
+
+        JPanel northPanel = new JPanel(new BorderLayout());
+
+        //BUG: button only pops up after it is clicked...
+        JButton updateButton = new JButton("Update");
+        northPanel.add(updateButton, BorderLayout.EAST);
+
```

Comment:
If you are going to default to Minneapolis then have the text field reflect the city you are showing